packages/babel-generator/src/generators/typescript.ts

```diff
@@ -475,14 +475,17 @@ export function TSLiteralType(this: Printer, node: t.TSLiteralType) {
   this.print(node.literal);
 }
 
-export function TSExpressionWithTypeArguments(
-  this: Printer,
-  node: t.TSExpressionWithTypeArguments,
-) {
+export function TSClassImplements(this: Printer, node: t.TSClassImplements) {
   this.print(node.expression);
   this.print(node.typeParameters);
 }
 
+export {
+  // TODO: Remove this in Babel 8
+  TSClassImplements as TSExpressionWithTypeArguments,
+  TSClassImplements as TSInterfaceHeritage,
+};
```

Comment:
```suggestion
if (!USE_ESM || !process.env.BABEL_8_BREAKING) {
  exports.TSExpressionWithTypeArguments = TSClassImplements;
  exports.TSInterfaceHeritage = TSClassImplements;
};
```

Does this work, or do we need them to be available in ESM Babel 7 builds?

Reply:
error TS7053: Element implicitly has an 'any' type because expression of type '"AnyTypeAnnotation" | "ArgumentPlaceholder" | "ArrayExpression" | "ArrayPattern" | "ArrayTypeAnnotation" | "ArrowFunctionExpression" | "AssignmentExpression" | "AssignmentPattern" | ... 242 more ... | "YieldExpression"' can't be used to index type 'Printer'.
  Property 'TSExpressionWithTypeArguments' does not exist on type 'Printer'.

ts will complain about this.